schema-registry/schema-registry-docker.sh

```diff
@@ -1,21 +1,32 @@
 #!/bin/bash
 
-: ${SCHEMA_REGISTRY_PORT:=8081}
-: ${SCHEMA_REGISTRY_KAFKASTORE_TOPIC:=_schemas}
-: ${SCHEMA_REGISTRY_KAFKASTORE_CONNECTION_URL:=$ZOOKEEPER_PORT_2181_TCP_ADDR:$ZOOKEEPER_PORT_2181_TCP_PORT}
-: ${SCHEMA_REGISTRY_DEBUG:=false}
+sr_cfg_file="/etc/schema-registry/schema-registry.properties"
 
-export SCHEMA_REGISTRY_PORT
-export SCHEMA_REGISTRY_KAFKASTORE_TOPIC
-export SCHEMA_REGISTRY_KAFKASTORE_CONNECTION_URL
-export SCHEMA_REGISTRY_DEBUG
+: ${SR_PORT:=8081}
```

Comment:
I think the older naming is clearer here, but more importantly, doesn't this lead to inconsistency when you name the container `schema-registry`, as is done in the docs (including the patched version in #9)? Or rather, when the `--link` is named that way? The auto-generated environment variables would be like `SCHEMA_REGISTRY_PORT_8081_TCP_ADDR`, wouldn't they?